adage/analyze/management/commands/import_gene_sample_expr.py

```diff
@@ -13,7 +13,7 @@
 
 For example, to load the sample-gene expression values for the organism
 "Pseudomonas aeruginosa" (whose taxonomy ID is 208964), the command will be:
-  python manage.py input_filename 208964
+  python manage.py import_gene_sample_expr input_filename 208964
```

Comment:
The input file that Jie sent to me is `data/all-pseudomonas-gene-normalized.pcl`. Although it is tab delimited, it doesn't have "tsv" extension, so I didn't put any extension in this example command either.